Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -434,15 +432,31 @@ bool IMixedRealityNearPointer.TryGetNearGraspPoint(out Vector3 position)
         /// <inheritdoc />
         bool IMixedRealityNearPointer.TryGetDistanceToNearestSurface(out float distance)
         {
-            distance = closestDistance;
-            return true;
+            if (closestProximityTouchable != null)
+            {
+                distance = closestDistance;
+                return true;
+            }
+            else
+            {
+                distance = 0.0f;
+                return false;
+            }
         }
 
         /// <inheritdoc />
         bool IMixedRealityNearPointer.TryGetNormalToNearestSurface(out Vector3 normal)
         {
-            normal = closestNormal;
-            return true;
+            if (closestProximityTouchable != null)
+            {
+                normal = closestNormal;
+                return true;
+            }
+            else
+            {
+                normal = Rotation * Vector3.forward;
```

Comment:
Mostly an optimization suggestion so we're not potentially duplicating math here: now that `closestNormal` is set to `Rotation * Vector3.forward;` up on line 214, it reads like any failure case which would lead to `closestProximityTouchable == null` would also lead to `closestNormal` staying equal to `Rotation * Vector3.forward` already. Could we just turn this into:

```c#
normal = closestNormal;
return closestProximityTouchable != null; // or return IsNearObject, to account for cases where that concept is overridden
```

Reply:
Done!